doc/common/styles/html/coqremote/footer.html

```diff
@@ -1,34 +1,431 @@
-    <div id="sidebarWrapper">
-      <div id="sidebar">
-        <div class="block">
-          <h2 class="title">Navigation</h2>
-          <div class="content">
-            <ul class="menu">
-              <li class="leaf">Standard Library
-                <ul class="menu">
-                  <li><a href="index.html">Table of contents</a></li>
-                  <li><a href="genindex.html">Index</a></li>
-                </ul>
-              </li>
-            </ul>
+
+          <!-- ************************
+            START STDLIB FOOTER 
+          ************************* -->
           </div>
         </div>
       </div>
     </div>
 
-  </div>
+  </main>
```

Comment:
This is a bit weird. This means that by default, the produced HTML will not be valid. We have to keep in mind that some people would like to view the doc locally. In this case, it is fine to not have any header / footer, but not to have broken HTML.

Reply:
@BastienSozeau

Reply:
Ah sorry, I didn't realize that this is closing tags that are opened in the header!